### PR TITLE
docs(API_GetConsoleIDs): add IconURL field

### PR DESCRIPTION
**Pull request by WedgeSparda**
Updated API docs to add new response parameter developed in this [PR](https://github.com/RetroAchievements/RAWeb/pull/2264/files)

### CODE REVIEW

**Review by wescopeland — DISMISSED** [review]
Thanks! I'll merge this when the RAWeb changes are released to production.

File: docs/v1/get-console-ids.md [hunk 1]
```diff
@@ -50,7 +52,13 @@ const consoleIds = await getConsoleIds(authorization);
 
 ```json [HTTP Response]
 [
-  { "ID": 1, "Name": "Mega Drive" }
+  {
+    "ID": 1,
+    "Name": "Mega Drive",
+    "IconURL": "https://static.retroachievements.org/assets/images/system/md.png",
+    "Active": true,
+    "IsGameSystem": true
```
Comment (wescopeland) [inline docs/v1/get-console-ids.md:60]: Is this and the `a` and `g` params part of a different change? Can we break this out to a different PR -- I'd like to merge the `IconURL` change when we release to prod today.

Comment (WedgeSparda) [reply docs/v1/get-console-ids.md:60]: Ah, yes, this is coming in this [PR](https://github.com/RetroAchievements/RAWeb/pull/2266). 
I forgot this PR was still opened and my latest change was added, sorry.

Comment (wescopeland) [reply docs/v1/get-console-ids.md:80]: No worries, once the additions are reverted I'll immediately merge this branch (and it will deploy). We'll probably deploy RAWeb to prod in an hour or two.

Comment (WedgeSparda) [reply docs/v1/get-console-ids.md]: Just reverted
